build.sbt

```diff
@@ -571,3 +570,12 @@ lazy val update2_12 = Seq(
     }
   }
 )
+
+lazy val xlint = Seq(
+  scalacOptions += {
+    CrossVersion.partialVersion(scalaVersion.value) match {
+      case Some((2, 12)) => "-Xlint:-unused,_"
```

Comment:
I got quite a few seemingly related to simulacrum:
>[error] /home/travis/build/typelevel/cats/core/src/main/scala/cats/Apply.scala:10: parameter value ev$macro$3 in method ap is never used
[error] @typeclass(excludeParents = List("ApplyArityFunctions"))